components/scripts/lib/info.sh

```diff
@@ -140,6 +140,8 @@ print_performance_characteristics() {
 
   print_realized_build_time_savings
 
+  print_potential_build_time_savings
+
```

Comment:
Thoughts on updating the `WARNING` to point out the `Potential build time savings`? 

![image](https://user-images.githubusercontent.com/5797900/210616417-a70bb681-372c-4485-8e91-505e35c9b612.png)

Reply:
Ha! I actually suggested tweaking this warning message slightly on one of the earlier PRs to use the "build time savings" language. Is that sort of what you have in mind?

Reply:
I would leave the warning text as it is since strictly speaking, we are reducing task exec time and that might not actually translate to build time savings depending on the parallelism of the build.